src/Mapping/Annotation/All.php

```diff
@@ -7,6 +7,11 @@
  * file that was distributed with this source code.
  */
 
+@trigger_error(sprintf(
+    'Requiring the file at "%s()" is deprecated since gedmo/doctrine-extensions 3.11, this file will be removed in version 4.0.',
+    __METHOD__
```

Comment:
How can we guarantee this scripts is always loaded inside a method?

Reply:
I'm not quite following here.

Including this file is not needed since Annotations 1.10 as it will fall back to using the autoloader, with support for manual requires in the AnnotationRegistry deprecated and removed in 2.0.  This package requires Annotations 1.13, so there isn't a need for supporting the older behavior.

To avoid unfixable deprecation notices, this file is no longer included anywhere in this package's code, with the notice being there if someone is loading it in their own code.

Reply:
I mean, we haven't a way to be sure this script was loaded from a method. In that case, the resulting message could be confusing. Check [this example](https://3v4l.org/Tl7FG).

Reply:
OH!  🤦 

That was supposed to be `__FILE__` but clearly I not only messed up the copy/paste, but couldn't even read the code right on your comment 😅 

It's fixed.